capstone/src/main/java/servlet/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event{
+    String id; 
+    String location;
+    String summary;
+
+    public void EventSummary(String summary, String id) {
+        this.summary = summary;
+        this.id = id;
+
```

Comment:
Delete line

Reply:
I'm still not clear on which line i should delete.

Reply:
Line 9 -- it's just unnecessary whitespace